docs/examples/ratify-verify-azure-cmd.md

```diff
@@ -1,296 +1,188 @@
 
-### Discover & Verify using Ratify
-
-1. Create a Ratify config with ORAS as the signature store and notary v2 as the signature verifier.
-Trust Policy reference: https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-policy
+2. This step above may take around 10 minutes to complete. The registration status can be checked by running the following command:
 
     ```bash
-    cat <<EOF > ~/.ratify/config.json 
-    { 
-        "store": { 
-            "version": "1.0.0", 
-            "plugins": [ 
-                { 
-                    "name": "oras"
-                }
-            ]
-        },
-        "policy": {
-            "version": "1.0.0",
-            "plugin": {
-                "name": "configPolicy",
-                "artifactVerificationPolicies": {
-                    "application/vnd.cncf.notary.signature": "any"
-                }
-            }
-        },
-        "verifier": {
-            "version": "1.0.0",
-            "plugins": [
-                {
-                    "name":"notaryv2",
-                    "artifactTypes" : "application/vnd.cncf.notary.signature",
-                    "verificationCerts": [
-                        "~/.config/notation/truststore"
-                    ],
-                    "trustPolicyDoc": {
-                        "version": "1.0",
-                        "trustPolicies": [
-                            {
-                                "name": "default",
-                                "registryScopes": [ "*" ],
-                                "signatureVerification": {
-                                    "level" : "strict" 
-                                },
-                                "trustStores": ["ca:certs"],
-                                "trustedIdentities": ["*"]
-                            }
-                        ]
-                    }
-                }
-            ]
-        }
-    }
-    EOF
+    az feature show --namespace "Microsoft.ContainerService" --name "EnableWorkloadIdentityPreview" -o table
+    Name                                                      RegistrationState
+    --------------------------------------------------------    -------------------
+    Microsoft.ContainerService/EnableWorkloadIdentityPreview    Registered
     ```
 
-2. Discover the signatures
+3. Establish federated identity credential. On AZ CLI `${RATIFY_NAMESPACE}` is where you deploy Ratify:
 
     ```bash
-    # Query for the signatures
-    export IMAGE_DIGEST_REF=$(docker image inspect $IMAGE | jq -r '.[0].RepoDigests[0]')
-    ratify discover -s $IMAGE_DIGEST_REF
+    az identity federated-credential create \
+    --name ratify-federated-credential \
+    --identity-name "${IDENTITY_NAME}" \
+    --resource-group "${GROUP_NAME}" \
+    --issuer "${AKS_OIDC_ISSUER}" \
+    --subject system:serviceaccount:"${RATIFY_NAMESPACE}":"gatekeeper-system"
```

Comment:
the `gatekeeper-system` is the service account name which might confuse users with the `gatekeeper-system` namespace. Wonder if we should explicitly describe it or change the name.

Reply:
Changed `gatekeeper-system` to `ratify-admin`, which aligns with https://github.com/deislabs/ratify/blob/main/docs/reference/oras-auth-provider.md#2-azure-workload-identity